docker-compose.yml

```diff
@@ -10,6 +10,9 @@ services:
       args:
         # This is used with TYPE="channel" to override the channel.
         CHANNEL: "${CHANNEL:-nightly}"
+
+        # Set network template file to future_template if it is not manually set to dev net.
+        NETWORK_TEMPLATE_PATH: "/tmp/network_config/${NETWORK_TEMPLATE:-future_template.json}"
```

Comment:
Let's change the default to `DevModeNetwork.json` instead

Reply:
@algochoi Sounds good - let's do it.